src/com/SocketCommunication.cpp

```diff
@@ -199,21 +183,14 @@ void SocketCommunication::requestConnection(std::string const &acceptorName,
   TRACE(acceptorName, requesterName);
   assertion(not isConnected());
 
-  std::string address;
-  const std::string addressFileName("." + requesterName + "-" + acceptorName + ".address");
+  ConnectionInfoReader conInfo(acceptorName, requesterName, _addressDirectory);
+  std::string address = conInfo.read();
```

Comment:
```suggestion
const std::string address = conInfo.read();
```
Could even be `const auto`

Reply:
Ok, about const. Don't like the `auto` here, using `string` makes it more clear to the developer what the actual type is.